src/test/kotlin/com/chimbori/crux/articles/GoldenFilesTest.kt

```diff
@@ -510,6 +514,22 @@ class GoldenFilesTest {
     }
   }
 
+  @Test
+  fun testNbc() {
+    fromFile(
+      "https://www.nbcnews.com/health/health-news/us-monkeypox-vaccines-testing-rcna37097",
+      "nbc.html"
+    ).run {
+      assertEquals("U.S. to ship 144,000 more monkeypox shots and expand testing as cases top 700", title)
+      assertStartsWith(
+        "The U.S. is stepping up its efforts to address the snowballing monkeypox outbreak — the country's largest to date by far, with 700 confirmed cases.",
+        document?.text()
+      )
+      assertEquals("Fri Jul 08 2022 00:44:04 GMT+0000 (UTC)", created)
```

Comment:
It would be good to have all dates be ISO-8601 formatted, that way callers don’t have to wonder which way to parse a date, or know about multiple formats.